MPSGameLift/Code/Include/MPSGameLift/IMatchmaking.h

```diff
@@ -10,6 +10,17 @@
 
 namespace MPSGameLift
 {
+    enum class MatchMakingFailReason
```

Comment:
Nit: Everything else in this file is using "Matchmaking", not "MatchMaking".
```suggestion
    enum class MatchmakingFailReason
```